src/test/java/com/coveo/spillway/SpillwayTest.java

```diff
@@ -118,20 +121,17 @@ public void simpleLimit() {
     assertThat(spillway.tryCall(john)).isFalse(); // Third tryCall fails
   }
 
-  @Test
-  public void multipleLimitsWithOverlap() {
+  @Test(expected = SpillwayLimitWithSameNameException.class)
+  public void multipleLimitsWithOverlap() throws Exception {
```

Comment:
This feels like a valid use case. I want to limit users to 1 call per second and 5 calls per hour.

Why not allow duplicate names in these situations?